tests/ImageSharp.Tests/TestUtilities/ImageProviders/FileProvider.cs

```diff
@@ -133,15 +133,14 @@ public override Image<TPixel> GetImage(IImageDecoder decoder)
             {
                 Guard.NotNull(decoder, nameof(decoder));
 
-                Key key = new Key(this.PixelType, this.FilePath, decoder);
+                if (!TestEnvironment.Is64BitProcess)
```

Comment:
Can't we keep caching smaller images on 32bit?
(By providing an upper megapixel limit.)

Reply:
It doesn't seem to affect our testing timings much so I'm inclined to keep it simple for now.